src/zos_files/zowe/zos_files_for_zowe_sdk/file_system.py

```diff
@@ -47,13 +45,17 @@ def create(self, file_system_name, options={}):
         """
         Create a z/OS UNIX zFS Filesystem.
 
-        Parameter
+        Parameters
         ---------
-        file_system_name: str - the name for the file system
+        file_system_name: str
+            the name for the file system
+        options: dict
+            specifies file system attributes
```

Comment:
I think we should use sentence case to make the descriptions consistent
```suggestion
        file_system_name: str
            Name of the file system
        options: dict
            Specifies file system attributes
```